templates/style.htm

```diff
@@ -0,0 +1,117 @@
+<style type="text/css">
```

Comment:
Rather than making this entire include file one large inline style block, please move the styles to a "styles.css" or some sort and use a stylesheet include instead.  This include file can than be used to apply multiple styles if necessary.